README.md

```diff
@@ -159,3 +159,25 @@ Once we are done using the modem object, it must be released using the `su_modem
 ```
 su_modem_destroy(modem);
 ```
+
+## Building and installing sigutils under Windows
+Under MSYS2 MinGW64
+First, you must create a build directory and configure it with:
+
+```
+% mkdir build
+% cd build
+% cmake -G "MSYS Makefiles" ..
```

Comment:
Reminder: specify the exact dependencies and the full command line (as MSYS2 tends to have different versions of the same tool in the PATH)